modules/sdk-core/src/bitgo/wallet/wallet.ts

```diff
@@ -3276,6 +3277,40 @@ export class Wallet implements IWallet {
     }
   }
 
+  /**
+   * Sends a transaction from a TSS (Threshold Signature Scheme) wallet.
+   *
+   * @param params
+   *    txRequestId - The ID of the transaction request.
+   *    walletPassphrase - The passphrase for the wallet.
+   *    isTxRequestFull - Flag indicating if the transaction request is full.
+   *    isTxRequestLite - Flag indicating if the transaction request is lite.
+   * @returns A promise that resolves to a SignedTransaction.
+   */
+  public async sendHotWalletAllTSSWithdrawalTransaction(
+    params: SendHotWalletAllTSSWithdrawalTransactionOptions
+  ): Promise<SignedTransaction> {
+    if (params.isTxRequestFull == params.isTxRequestLite) {
```

Comment:
why do we need both isTxRequestFull and isTxRequestLite? only one should be enough.

Reply:
Fixed!